src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/RangeFilterCRUDTest.java

```diff
@@ -1475,7 +1470,7 @@ class invalidSpecializationShouldNotChangeDatabaseContents {
 
                     @Test
                     @DisplayName("Invalid specialization name change should not change database contents")
-                    void invalidSpecializationNameChangeShouldNotChangeDatabaseContents() throws SQLException, InvalidFilterTypeException {
+                    void invalidSpecializationNameChangeShouldNotChangeDatabaseContents() throws SQLException, RangeFilterException {
```

Comment:
Does consistency matter? in every other test the rangefilterexception is written first

Reply:
nvm, not worth the time